config/config-defaults.yaml

```diff
@@ -87,6 +87,11 @@ data:
     # no default-resolver-type is specified by default
     default-resolver-type:
 
+    # default-imagepullbackoff-timeout contains the default duration to wait
+    # before requeuing the TaskRun to retry, specifying 0 here is equivalent to fail fast
+    # possible values could be 1m, 5m, 10s, 1h, etc
+    # default-imagepullbackoff-timeout: "5m"
```

Comment:
Are we expecting make this effective in this PR?

Reply:
This is a configuration for the end users. The operators can choose to set it to any value depending on their infrastructure and needs. By default, Tekton pipelines does not change and does mark `imagePullBackOff` as a permanent error (just like before).